src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,147 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+
+import duke.command.AddCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.ExitCommand;
+import duke.command.ListCommand;
+import duke.command.MarkCommand;
+import duke.command.UnmarkCommand;
+
+public class ParserTest {
+
+    @Test
+    public void testParseByeCommand() {
```

Comment:
I think test method names should be formatted as `featureUnderTest_testScenario_expectedBehavior()`?